pages/blog/websocket-part1.md

```diff
@@ -0,0 +1,495 @@
+---
+title: WebSocket, Shrek, AsyncAPI - An Opinionated Intro
+date: 2021-04-09T06:00:00+01:00
+type: Communication
+tags:
+  - websocket
+cover: /img/posts/websocket-part1/cover.webp
+authors:
+  - name: Lukasz Gornicki
+    photo: /img/avatars/lpgornicki.webp
+    link: https://twitter.com/derberq
+    byline: AsyncAPI Maintainer and Community Guardian
+excerpt: Websocket is a protocol, an industry standard for building client applications that users love to use. What AsyncAPI has to do with it?
+featured: true
+---
+
+It is a pretty subjective post. I share my perspective, taking into account years of experience building backend and frontend with user experience in mind. 
+
+>  Everything we hear is an opinion, not a fact. Everything we see is a perspective, not the truth.
+― [Marcus Aurelius](https://www.politifact.com/factchecks/2019/sep/26/viral-image/no-marcus-aurelius-didnt-say-about-opinions-and-fa/)
+
+This blog post is the first one about Websocket, but there will be more. I'm working on a series of blog posts about WebSocket.
+
+## What is WebSocket
+
+It is a pretty old protocol used for duplex communication over TCP connection. It was standardized in 2011. Yes, ten years ago means it is old, super old. 
+
+So why are we even mention it in 2021? 
```

Comment:
Shouldn't it be "So why are we even mentioning it in 2021?" ?

Reply:
how about `So why do we even mention it in 2021?` ?